plugin-packs/postcss-preset-env/test/basic.ff66.expect.css

```diff
@@ -320,52 +312,35 @@
 }
 
 .color-function {
-	prop-1: rgb(0,132,94);
+	prop-1: rgb(0, 132, 94);
 	prop-2: 'color(display-p3 0.02472 0.01150 0.00574 / 1)';
-	prop-3: rgba(7,3,1,1);
-	prop-4: rgba(7,3,1,calc(33 / 22));
-	prop-5: rgb(255,255,255);
+	prop-3: rgb(7, 3, 1);
+	prop-4: rgb(7, 3, 1);
+	prop-5: color(display-p3 1 1 1 1);
 }
 
 .oklab {
 	color-1: rgb(73, 71, 69);
-	color-1: color(display-p3 0.28515 0.27983 0.27246);
-	color-2: rgba(121, 34, 67, 1);
-	color-2: color(display-p3 0.43853 0.16014 0.26019 / 1);
-	color-3: rgba(121, 34, 67, .5);
-	color-3: color(display-p3 0.43853 0.16014 0.26019 / .5);
-	color-4: rgba(121, 34, 67, 1);
-	color-4: color(display-p3 0.43853 0.16014 0.26019 / 100%);
+	color-2: rgb(121, 34, 67);
+	color-3: rgba(121, 34, 67, 0.5);
+	color-4: rgb(121, 34, 67);
```

Comment:
The updated serializer inserts spaces at consistent positions and normalizes values. (e.g. `.5` -> `0.5`)